### DISCUSSION TITLE
BANANAPI系列会导致编译失败

### DISCUSSION THREAD

**Discussion by VIKINGYFY** [immortalwrt/immortalwrt#1418] [Q&A]
### Describe the bug

不确定是固件大小超限还是其他原因

### ImmortalWrt version

master

### ImmortalWrt release

57417cd9a39564a56f1dfb9ec5f6e6d9b46c2e94

### ImmortalWrt target/subtarget

MTK

### Device

BANANAPI

### Image kind

Self-built image

### Steps to reproduce

云编译日志
[logs_27049693746.zip](https://github.com/user-attachments/files/16582901/logs_27049693746.zip)

### Terms

- [X] I am reporting an issue for ImmortalWrt, not an unsupported fork.

**Comment by 1715173329** [comment]
对。

**Reply by VIKINGYFY** [reply]
> 对。

啊，意思是大小超限？那不太行啊，插件就hp、ts、wol、upnp这四个
